doc/release-notes/5.13-release-notes.md

```diff
@@ -0,0 +1,243 @@
+
+4\. Deploy this version.
+
+- `$PAYARA/bin/asadmin deploy dataverse-5.13.war`
+
+5\. Restart Payara
+
+- `service payara stop`
+- `service payara start`
+
+6\. Reload citation metadata block
+
+   `wget https://github.com/IQSS/dataverse/releases/download/v5.11/citation.tsv`
+   `curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @citation.tsv -H "Content-type: text/tab-separated-values"`
+- Add the updated citation.properties file to the appropriate directory
+
+7\. Replace Solr schema.xml to allow multiple production locations and support for geospatial indexing to be used. See specific instructions below for those installations without custom metadata blocks (1a) and those with  custom metadata blocks  (1b).
+
+7a\.
+
+For installations without Custom Metadata Blocks:
+
+-stop solr instance (usually service solr stop, depending on solr installation/OS, see the [Installation Guide](https://guides.dataverse.org/en/5.13/installation/prerequisites.html#solr-init-script)
+
+-replace schema.xml
+
+cp /tmp/dvinstall/schema.xml /usr/local/solr/solr-8.11.1/server/solr/collection1/conf
+
+-start solr instance (usually service solr start, depending on solr/OS)
+
+
+7b\. 
+
+For installations with Custom Metadata Blocks:
+
+-stop solr instance (usually service solr stop, depending on solr installation/OS, see the [Installation Guide](https://guides.dataverse.org/en/5.13/installation/prerequisites.html#solr-init-script)
+
+- edit the following line to your schema.xml (to indicate that productionPlace is now multiValued='true"):
+
+    `<field name="productionPlace" type="string" stored="true" indexed="true" multiValued="true"/>`
+
+- add the following lines to your schema.xml to add support for geospatial indexing:
+
+    `<!-- Dataverse geospatial search -->`
+    `<!-- https://solr.apache.org/guide/8_11/spatial-search.html#rpt -->`
+    `<field name="geolocation" type="location_rpt" multiValued="true" stored="true" indexed="true"/>`
+    `<!-- https://solr.apache.org/guide/8_11/spatial-search.html#bboxfield -->`
+    `<field name="boundingBox" type="bbox" multiValued="true" stored="true" indexed="true"/>`
+    `<!-- Dataverse - per GeoBlacklight, adding field type for bboxField that enables, among other things, overlap ratio calculations -->`
+    `<fieldType name="bbox" class="solr.BBoxField"
+           geo="true" distanceUnits="kilometers" numberType="pdouble" />`
+
+- restart solr instance (usually service solr start, depending on solr/OS)
+
+### Optional Upgrade Step: remove Workflow Schema fields from Solr index
+
+In Dataverse 5.12 we added a new experimental metadata schema block for workflow deposition.
+We included the fields within the standard Solr schema we provide. With this version, we
+removed it from the schema. If you are deploying the block to your installation, make sure to
+update your index.
+
+If you already added these fields, you can delete them from your index when not using the schema.
```

Comment:
Also, could somebody please rewrite/clarify this sentence. If I understand it correctly (a big _if_), the words "delete from your index" in it mean "delete from your Solr schema", and then "when not using the schema" really means "if you are not using this metadata block". It did confuse me, because we usually don't call metadata blocks "schemas", and because we are talking about solr schema in the same paragraph... 
I am kinda slow of course... but I can see how it may confuse another admin - ?